src/searchForm/index.tsx

```diff
@@ -17,6 +17,9 @@ const primaryFieldsWidth = '240px'
 const secondaryFieldsWidth = '160px'
 const buttonHeight = '48px'
 const submitButtonWidth = '88px'
+const positionAutocompleteFrom = '-8px'
+const positionAutocompleteTo = '268px'
```

Comment:
What are these magic values ?

Reply:
We should either:
 - compute them from some other CSS values (to understand tier value)
 - or compute them using some JS
 - or heavily document what's going on

Reply:
Documented in sketch, according to the grid they use for the form (which I think it's the same as @gabrielagabriel showed us. So maybe there's a need to have a grid, where we could also retrieve the left position of each column.